packages/cms-cli/commands/auth.js

```diff
@@ -22,58 +22,60 @@ const {
 } = require('../lib/usageTracking');
 const { PORTAL_ID, CLIENT_ID, CLIENT_SECRET } = require('../lib/prompts');
 
-const getAuthContext = async () => {
+const getAuthContext = () => {
   const prompt = inquirer.createPromptModule();
   return prompt([PORTAL_ID, CLIENT_ID, CLIENT_SECRET]);
 };
 
 const COMMAND_NAME = 'auth';
 
+async function authAction(type, options) {
```

Comment:
I think that we should move this to a separate file outside of the `commands` and that we should adjust the code so that we're not doing stuff that only makes sense in the context of the auth command. For example, `--config` is an option just of `hs auth`. Maybe the solution is to make a function that is just being used to go through the prompted oauth flow.

Reply:
I've updated the code and extracted some of the oauth-specific logic into a separate oauth file, but I am still exporting a method from `auth` that I am using within `init` -- not sure if this is what you meant or not.